ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -2863,6 +2868,9 @@ private static boolean isLockableTable(Table t) {
 
         case TABLE:
           t = input.getTable();
+          if (!fullTableLock.contains(t)) {
```

Comment:
Is the table.equals will work correctly here, as expected? I have seen another PR where we started to create special equals method for table comparison: https://github.com/apache/hive/pull/1095 - might be conflicting later

Reply:
table equals compares table names, so should be Ok